src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PAYMENT = 40;
+  const LONG_TERM_RENTAL_DAYS = 7;
+  const SHORT_TERM_RENTAL_DAYS = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  let sum = days * PAYMENT;
+
+  if (days >= LONG_TERM_RENTAL_DAYS) {
+    sum -= LONG_TERM_DISCOUNT;
+
+    return sum;
```

Comment:
```suggestion
    return sum -= LONG_TERM_DISCOUNT;
```
If you just return instead you can use const

Reply:
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/172060245/2de5d386-d7e5-40ec-9c56-1cbeb68e98e4)
Still doesn't work( or maybe I should do smth else?